src/components/FormElement.tsx

```diff
@@ -1,14 +1,25 @@
 import type {ForwardedRef} from 'react';
-import React, {forwardRef} from 'react';
+import React, {forwardRef, useRef} from 'react';
 import type {ViewProps} from 'react-native';
 import {View} from 'react-native';
+import useFormSetup from '@hooks/useFormSetup';
 import * as ComponentUtils from '@libs/ComponentUtils';
 
-function FormElement(props: ViewProps, ref: ForwardedRef<View>) {
+function FormElement(props: ViewProps, outerRef: ForwardedRef<View>) {
+    const formRef = useRef<View | null>(null);
+    useFormSetup(formRef.current);
     return (
         <View
             role={ComponentUtils.ACCESSIBILITY_ROLE_FORM}
-            ref={ref}
+            ref={(ref) => {
+                formRef.current = ref;
+                if (typeof outerRef === 'function') {
+                    outerRef(ref);
+                } else if (outerRef && 'current' in outerRef) {
+                    // eslint-disable-next-line no-param-reassign
+                    outerRef.current = ref;
+                }
```

Comment:
Would you de-duplicate this code with...

https://github.com/Expensify/App/blob/00fc955b129c572d79fb0a862359544799adffd3/src/libs/mergeRefs.ts#L10-L15

Reply:
From our perspective, a utility like...

```ts
function interceptRef<T>(targetRef: MutableRef<T | null>): Ref<T> {
  // ...
}
```

...used like...

```ts
<View
  role={ComponentUtils.ACCESSIBILITY_ROLE_FORM}
  ref={interceptRef(formRef)}
/>
```

...would be cool. I think it could be used as-is in 10 places in the codebase or more, deduplicating similar snippets. Big-scale de-duplication is not necessary in the scope of this PR, though.